crates/ironrdp-acceptor/src/connection.rs

```diff
@@ -438,6 +521,22 @@ impl Sequence for Acceptor {
 
                 debug!(message = ?client_info, "Received");
 
+                if !protocol.intersects(nego::SecurityProtocol::HYBRID | nego::SecurityProtocol::HYBRID_EX) {
+                    let creds = client_info.client_info.credentials;
+
+                    if self.creds.as_ref().map_or(true, |srv_creds| srv_creds != &creds) {
+                        // how authorization should be denied with standard RDP security?
```

Comment:
nitpick: Maybe we should emphasize this way?
```suggestion
                        // FIXME: How authorization should be denied with standard RDP security?
                        // Since standard RDP security is not a priority, we just send a ServerDeniedConnection ServerSetErrorInfo PDU.
```